docs/prometheus-examples.md

```diff
@@ -0,0 +1,37 @@
+## Prometheus examples
+A basic Prometheus setup is described in [prometheus-quickstart.md](prometheus-quickstart.md).  This document provides examples for how a permanent Prometheus setup can be deployed.  It is assumed you have some familiarity with Linux and Systemd.  As these are all examples, they obviously need not be followed verbatim.  Samples are provided as-is with minimum necessary descriptions. You should refer to official documentation to fully understand them. Please find the following samples in the `docs/samples` directory:
+ - alerting.rules.yml
+ - alertmanager.service
+ - alertmanager.yml
+ - blackbox_exporter.service
+ - blackbox.yml
+ - prometheus.service
+ - prometheus.yml
+
+### Voting node health
+This `alerting.rules.yml` example has two alerts that can detect potentially transient failures. For example, our alerts will catch these problems:
+ - The node appears online, but `dcrd` has stalled (BlockHeightStalled)
+ - Block height is correct but `dcrwallet` doesn't agree with the other voting nodes (LiveTicketParity)
+
+On my testnet vsp, which uses extremely cheap virtual servers, the first alert frequently catches problems early.  The second scenario happens often, but usually resolves itself within a few minutes. So that alert is configured to wait for a much longer duration before firing.  When the duration is tuned appropriately, these alerts can inform you of a problem before `dcrstakepool` itself collapses.  At a minimum, when you find your `dcrstakepool` instance offline, you can pull up the Prometheus `/alerts` page and drill down to see which specific nodes aren't healthy.
+ 
+### Systemd service units
+These [service units](https://www.freedesktop.org/wiki/Software/systemd/) assume you've set up Prometheus under various `/opt` subdirectories, with binaries stored in `/opt/bin`.  They use an unprivileged user account to run the daemon. For your convenience, `ExecReload` has been configured so that you can reload configuration on the fly, e.g. 	`systemctl reload prometheus`.  Assuming you went through the quickstart already, here is how you could set up the `prometheus` daemon :
+```bash
+ sudo mkdir -p /opt/prometheus
+ sudo cp -r prometheus-2.13.1.linux-amd64/{consoles,console_libraries} /opt/prometheus/
+ sudo cp prometheus-2.13.1.linux-amd64/prometheus /opt/bin/
```

Comment:
Starting on a clean ubuntu, I also had to create the /opt/bin directory.

Also the service requires the promtool binary as well. Also, the previously made prometheus.yml and alerting.rules.yml needs to be in the prometheus folder. If I was following your quickstart guide, then it's already in the user's folder, you could just change `sudo cp -r prometheus-2.13.1.linux-amd64/{consoles,console_libraries}` to have bash completion `*` at the end to simplify and take everything.

I agree that the person setting this up should be able to debug this, so it's up to you if this is intentionally left out for brevity.